### PR TITLE
Delete Exactly-About.info.xml & Create GoNewSoft.com.xml

### PR DESCRIPTION
**Pull request by MojtabaFaal**
Delete Exactly-About.info.xml
Old Website

&

Create GoNewSoft.com.xml
New Website

### CODE REVIEW

File: src/chrome/content/rules/GoNewSoft.com.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="GoNewSoft.com">
+        <target host="gonewsoft.com" />
+        <target host="www.gonewsoft.com" />
+		
+		<rule from="^http://(?:www\.)?gonewsoft\.com/" to="https://gonewsoft.com/"/>
```
Comment (jsha) [inline src/chrome/content/rules/GoNewSoft.com.xml]: This rule removes the initial www but it is not necessary. You can instead use:

```
<rule from="^http:"
  to="https:" />
```

Comment (MojtabaFaal) [inline src/chrome/content/rules/GoNewSoft.com.xml]: I would like to be removed www

for example :
http://GoNewSoft.com
http://www.GoNewSoft.com
https://www.GoNewSoft.com

Redirect to :
https://GoNewSoft.com

Comment (jsha) [inline src/chrome/content/rules/GoNewSoft.com.xml]: Yes. But with the simple rule I suggested, visitors will go first to https://www.gonewsoft.com/, and then be redirected to https://gonewsoft.com/ by the server. We tend to prefer that style, unless the www version doesn't have a certificate.

**Comment by MojtabaFaal** [comment]
Better?

**Comment by jsha** [comment]
Great, thanks!

**Pull request merged**